mod-password-validator/src/test/resources/domain/basic.feature

```diff
@@ -0,0 +1,29 @@
+Feature: mod-password-validator integration tests
```

Comment:
You duplicate permissions and calling destroy data(you call in `ModPasswordValidatorApiTest`). Please, remove this file and just add run feature (domain/basic.feature) in the `ModPasswordValidatorApiTest`. For example you can see from another modules: ModNotesTest and QuickMarcApiTest